README.md

```diff
@@ -38,73 +39,132 @@ just install Ketrew:
   with OpenSSL (package `ssl`) or [nqsb-TLS](https://nqsb.io/) (package
   `tls`, *experimental*).
 
-This gets you the `ketrew` executable and the `ketrew_pure` and `ketrew`
-libraries.
+This gets you the
 
-Then, at runtime, you need the `ssh` client in the `$PATH`.
+  - `ketrew` executable that can be used to schedule and run workflows.
+  - [ketrew_pure](http://seb.mondet.org/software/ketrew/api/Ketrew_pure.html)
+    library, that represents the [stateless]
+    (https://en.wikipedia.org/wiki/Side_effect_%28computer_science%29)
+    part of describing workflows.
+  - [ketrew](http://seb.mondet.org/software/ketrew/api/Ketrew.html) library,
+    that handles the messy orchestration of those tasks and, significantly,
+    has the [EDSL](http://seb.mondet.org/software/ketrew/api/Ketrew.EDSL.html)
+    to write workflows.
```

Comment:
Getting some insight into the internals helps a lot with getting started, otherwise it's an impenetrable black box of a novel DSL and its unseen interpretation.